src/c_api/c_api_symbolic.cc

```diff
@@ -75,6 +78,10 @@ int MXListAllOpNames(nn_uint *out_size,
                      const char ***out_array) {
   mxnet::op::RegisterLegacyOpProp();
   mxnet::op::RegisterLegacyNDFunc();
+#if MXNET_USE_NGRAPH == 1
+  // ngraph imperative interface
+  ngraph_bridge::InitImperative();
```

Comment:
this seems to be an ad-hoc place. Is there a better way to initialize ngraph? @piiswrong

Reply:
Would this place work? https://github.com/apache/incubator-mxnet/blob/master/src/initialize.cc#L45

Reply:
We're following the example of the Legacy ops above, this function is effectively doing the same thing with nGraph to support Gluon. We could rename it `RegisterNGraphOps` if that makes it clearer?